tests/unit/navigator.test.ts

```diff
@@ -0,0 +1,9 @@
+import defaultImport from "navigator";
+import * as namedImport from "navigator";
+
+describe("navigator.userAgent", () => {
+  it("should have a navigator userAgent", () => {
+    expect(defaultImport.userAgent).toEqual(navigator.userAgent);
```

Comment:
should we maybe do a regex check here or starts with "llrt/"?

Reply:
Thanks for pointing that out.
According to the specification, it is not required to start with a slash, but we will follow the implementation and make sure that it starts with "llrt/".

Reply:
Ok if that's not the case they we should revert to `llrt {version}` because this is already used for stats :)

Reply:
OK, I understand.
I'll try to revert to `llrt {version}`.